src/server/lib/format.js

```diff
@@ -13,32 +16,39 @@ const EXPENDITURE_CATEGORIES = {
   ec7: '7-Administrative and Other'
 }
 
-function currency (value, optional = false) {
-  if (optional && value == null) {
-    return value
-  }
+/**
+ * Normalize casing of single word values.
+ * This is especially useful for "Yes"/"No" responses.
+ *
+ * @param {string} value
+ * @returns {string}
+ */
+function capitalize (value) {
+  if (value == null) return value
+  return `${value[0].toUpperCase()}${value.slice(1).toLowerCase()}`
+}
+
+function currency (value) {
+  if (value == null) return value
   return round(value, 2).toString()
 }
 
 function ec (value) {
   return EXPENDITURE_CATEGORIES[value]
 }
 
-function zip (value, optional = false) {
-  if (optional && value == null) {
-    return value
-  }
+function zip (value) {
```

Comment:
@ajhyndman  Did you mean to include all these optional/required changes in this PR that only mentions the Yes/No capitalization?

Reply:
Ah, yes.  Sorry, I included a small refactor to simplify things.

https://github.com/usdigitalresponse/arpa-reporter/pull/400/commits/231231de5ceb7036e5690ce1c8a7c5765d595f03